packages/typespec-autorest-canonical/src/decorators.ts

```diff
@@ -0,0 +1,28 @@
+import { Program, Service, Type } from "@typespec/compiler";
+import { createStateSymbol } from "./lib.js";
+
+export const namespace = "AutorestCanonical";
+
+/**
+ * Parameters that may be passed to a RefProducer function.  Specific RefProducer
+ * functions may define additional parameters.
+ */
+export interface RefProducerParams {
```

Comment:
This one too there is nothing that this getter will do

Reply:
removed